wagtail/blocks/base.py

```diff
@@ -274,26 +273,59 @@ def render(self, value, context=None):
         return mark_safe(render_to_string(template, new_context))
 
     def get_preview_context(self, value, parent_context=None):
-        # We do not fall back to `get_context` here, because the preview context
-        # will be used for a complete view, not just the block. Instead, the
-        # default preview context uses `{% include_block %}`, which will use
-        # `get_context`.
+        """
+        Return a dict of context variables to be used as the template context
+        when rendering the block's preview. The ``value`` argument is the value
+        returned by :meth:`get_preview_value`. The ``parent_context`` argument
+        contains the following variables:
+
+        - ``request``: The current request object.
+        - ``block_def``: The block instance.
+        - ``block_class``: The block class.
+        - ``bound_block``: A ``BoundBlock`` instance representing the block and its value.
```

Comment:
The `page_title` variable is intentionally undocumented as it's not that relevant for configuring the preview, and may be confused for `Page.title`.